packages/meter/package.json

```
@@ -63,7 +64,7 @@
         "@spectrum-web-components/shared": "^0.49.0"
     },
     "devDependencies": {
-        "@spectrum-css/progressbar": "^4.1.5"
+        "@spectrum-css/progressbar": "^5.0.0-s2-foundations.19"
```

```suggestion
        "@spectrum-css/meter": "^0.0.0-s2-foundations.2",
        "@spectrum-css/progressbar": "^5.0.0-s2-foundations.19",
```

This component is not loading the meter styles which are now separated from progressbar.

---

This is tracked in this [PR](https://github.com/adobe/spectrum-web-components/pull/4835)